oss-internship-2020/openjpeg/examples/convert_helper.h

```diff
@@ -0,0 +1,8 @@
+// imagetopnm and the two functions it calls internaly are copied from the
```

Comment:
Would it also be possible to add a helper library to the `CMakeLists.txt` that just builds the library's `convert.c`? This would avoid the copy.

Generally, we try to avoid to ship code in SAPI that was copied from somewhere else. At the very least, this makes our lifes easier with respect to licensing. In this case it's not too bad, because the copied code is BSD, but it's good practice to avoid copies :)